components/infobox/commons/infobox_company.lua

```diff
@@ -137,11 +137,11 @@ end
 function Company:_getOrganizerPrizepools()
 	local prizemoney = mw.ext.LiquipediaDB.lpdb('tournament', {
 		conditions =
-			'[[organizers_organizer1::' .. self.pagename .. ']] OR ' ..
-			'[[organizers_organizer2::' .. self.pagename .. ']] OR ' ..
-			'[[organizers_organizer3::' .. self.pagename .. ']] OR ' ..
-			'[[organizers_organizer4::' .. self.pagename .. ']] OR ' ..
-			'[[organizers_organizer5::' .. self.pagename .. ']]',
+			'[[organizers_organizer1::' .. self.pagename:gsub('_', ' ') .. ']] OR ' ..
```

Comment:
imo we should store with underscores instead of having to do the gsubs here (since pagenames in general should be stored with underscores)
for the transition phase maybe query for both options?